deploy/install/01-prerequisite/04-default-setting.yaml

```diff
@@ -5,42 +5,3 @@ metadata:
   namespace: longhorn-system
 data:
   default-setting.yaml: |-
-    backup-target:
```

Comment:
Can we comment them out instead of removing them for reference purpose? 

note: it's fine we have empty default-setting.yaml in the deploy.yaml in longhorn/longhorn, because that is automatically generated instead.

Reply:
Addressed.